File: src/test/java/org/simpleframework/xml/transform/net/URLTransformTest.java
package org.simpleframework.xml.transform.net;

import java.net.URL;
import junit.framework.TestCase;

public class URLTransformTest extends TestCase {
   
   public void testURL() throws Exception {
      URL file = new URL("http://www.google.com/");
      URLTransform format = new URLTransform();
      String value = format.write(file);
      URL copy = format.read(value);
      
      assertEquals(file, copy);      
   }
}

File: port/src/main/Xml/Transform/URLTransformTest.cs
#region Using directives
using SimpleFramework.Xml.Transform;
using SimpleFramework.Xml;
using System;
#endregion
package SimpleFramework.Xml.transform;
public class URLTransformTest : TestCase {
   public void TestURL() {
      URL file = new URL("http://www.google.com/");
      URLTransform format = new URLTransform();
      String value = format.write(file);
      URL copy = format.read(value);
      AssertEquals(file, copy);
   }
}
}
